In [11]:
import numpy as np
import matplotlib.pyplot as plt

angle_data = np.loadtxt('angle.txt') #,skiprows=1)
angle_data = np.array(angle_data)



In [12]:
np.max(angle_data, axis=0)

array([1.799990e+05, 5.619180e-01, 5.746090e-01, 3.141558e+00])

In [13]:
np.min(angle_data, axis=0)

array([ 1.      , -0.574309, -0.414788, -3.1415  ])

In [14]:
np.mean(angle_data, axis=0)

array([ 9.00000000e+04,  1.27958094e-03,  2.06202583e-03, -8.00701559e-02])

In [15]:
np.std(angle_data, axis=0)

array([5.19612356e+04, 1.22703222e-01, 1.26236844e-01, 1.74984095e+00])

In [8]:
angle_data = np.array(angle_data[1000:,:])
angle_data.shape

(177999, 4)

In [9]:
np.max(angle_data, axis=0), np.min(angle_data, axis=0), np.mean(angle_data, axis=0)

(array([1.799990e+05, 5.619180e-01, 5.746090e-01, 3.141558e+00]),
 array([ 2.00100e+03, -5.74309e-01, -4.14788e-01, -3.14150e+00]),
 array([ 9.10000000e+04,  1.31268012e-03,  2.05342910e-03, -8.09647313e-02]))

In [10]:
np.std(angle_data, axis=0)

array([5.13838853e+04, 1.23368881e-01, 1.26934381e-01, 1.75962352e+00])


## Thus we find out we should normalize the z-axis by dividing by 1.75 (or 2)